.github/workflows/test.yml

```
@@ -441,12 +441,12 @@ jobs:
     timeout-minutes: 220
     strategy:
       fail-fast: false
-      matrix:
-        # Valid options are:
+        matrix:
```

actionlint (1.7.12)

[error] could not parse as YAML: mapping values are not allowed in this context

(syntax-check)